hugo/content/about.md

```diff
@@ -0,0 +1,73 @@
+---
+title: About
+toc: true
+---
+
+# About the project
+
+Mumble is a free, open source, low latency, high quality voice chat application.
+
+## Source Code & Development
+
+You find the Source Code and further instructions regarding Building and Development in our [GitHub Repo](https://github.com/mumble-voip).
+
+## Licenses
+
+<!-- https://github.com/mumble-voip/mumble/blob/master/LICENSE -->
+
+<!-- more licenses? should be as correct and complete as possible -->
```

Comment:
I think we don't need to specify licenses here explicitly. If you think it's important you can state something like "We are generally using Open Source license without copy-left clauses (e.g. BSD-3)" but in the end the user can simply checkout the license for the individual projects.

Reply:
Then I will link to Github and tell people to look at the licenses of each project.
Ok?

Reply:
Sounds good